src/ui/MessageContent/index.tsx

```diff
@@ -332,6 +332,7 @@ export default function MessageContent(props: MessageContentProps): ReactElement
                     setShowFeedbackModal(true);
                   }
                 }}
+                disabled={message?.myFeedback?.rating === FeedbackRating.BAD}
```

Comment:
If the rating options exceed three, it seems it might cause issues with the lower version.
What do you think about handling it this way?

`disabled={message.myFeedback && message.myFeedback.rating !== FeedbackRating.GOOD}`

Reply:
yes definitely, thanks! i applied this suggestion